static-build/qemu/Dockerfile

```diff
@@ -0,0 +1,45 @@
+from ubuntu:xenial
```

Comment:
Nit: might be clearer to write `from ubuntu:16.04` as folk tend to remember the numeric convention for LTS releases but not always recall the codenames ;)

Reply:
done